src/tss2-esys/esys_iutil.c

```diff
@@ -1037,7 +1039,7 @@ iesys_check_sequence_async(ESYS_CONTEXT * esys_context)
         LOG_ERROR("Esys called in bad sequence.");
         return TSS2_ESYS_RC_BAD_SEQUENCE;
     }
-    /* TODO: Check if RESUBMISSION BELONGS HERE OR IN THE FINISH METHOD. */
+//TODO: Check if RESUBMISSION BELONGS HERE OR RATHER INTO THE FINISH METHOD.
```

Comment:
That was changed in 0a7027c2, why do we want to change it back?

Reply:
Sorry for changing back by mistake. Updated